gapic/packaging/dependencies.yaml

```diff
@@ -49,7 +49,11 @@ gax_version:
   ruby:
     lower: '0.8.0'
   java:
-    lower: '0.6.1'
+    lower: '1.2.0'
+
+gax_grpc_version:
```

Comment:
Hmm well gax 1.2.0 is incomplete, so someone using only this shouldn't be able to build a coherent package, right?